dashboard/app/controllers/courses_controller.rb

```diff
@@ -3,9 +3,16 @@ class CoursesController < ApplicationController
 
   before_action :require_levelbuilder_mode, except: [:index, :show, :vocab, :resources, :code, :standards]
   before_action :authenticate_user!, except: [:index, :show, :vocab, :resources, :code, :standards]
+  before_action :set_unit_group, only: [:show, :vocab, :resources, :code, :standards, :edit, :update, :get_rollup_resources]
```

Comment:
Do you feel like the tests that are already here sufficiently cover this and other new functionaliity?

Reply:
Ya I think so. There is a good amount of testing in the courses_controller_test.rb for these actions